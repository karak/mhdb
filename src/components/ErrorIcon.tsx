import * as React from 'react';
import * as ReactDOM from 'react-dom';
import { red500 } from 'material-ui/styles/colors';
import FontIcon from 'material-ui/FontIcon';

// tslint:disable-next-line:function-name
export default function ErrorIcon() {
  return (
    <FontIcon className="material-icons" color={red500}>
      error
    </FontIcon>
  );
}
