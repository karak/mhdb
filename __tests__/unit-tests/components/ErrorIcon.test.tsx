import '../../helpers/setup';
import * as React from 'react';
import ErrorIcon from '../../../src/components/ErrorIcon';
import { shallow } from 'enzyme';
import { default as toJson } from 'enzyme-to-json';

describe('ErrorIcon', () => {
  it('matches with snapshot', () => {
    const tree = toJson(shallow(<ErrorIcon />));

    expect(tree).toMatchSnapshot();
  });
});
