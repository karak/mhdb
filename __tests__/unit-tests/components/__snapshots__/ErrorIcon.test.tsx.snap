// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorIcon matches with snapshot 1`] = `
<FontIcon
  className="material-icons"
  color="#f44336"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  error
</FontIcon>
`;
